primitives/src/block.rs

```diff
@@ -365,11 +365,89 @@ mod tests {
     fn version_is_signalling() {
         let version = Version::from_consensus(0b00100000000000000000000000000010);
         assert!(Version::is_signalling_soft_fork(&version, 1));
+        let version = Version::from_consensus(0b00110000000000000000000000000000);
+        assert!(Version::is_signalling_soft_fork(&version, 28));
     }
 
     #[test]
     fn version_is_not_signalling() {
         let version = Version::from_consensus(0b00100000000000000000000000000010);
         assert!(!Version::is_signalling_soft_fork(&version, 0));
     }
+
+    #[test]
+    fn version_to_consensus() {
+        let version = Version::from_consensus(1234567890);
+        assert_eq!(version.to_consensus(), 1234567890);
+    }
+
+    // Test the creation of a dummy block
+    #[cfg(feature = "alloc")]
+    #[test]
+    fn block_creation() {
```

Comment:
The test comment is confusing because it makes me ask 'why would we test the creation of a dummy block?' Also the test name `block_creation` means exactly "test block creation". In general we only comment tests if its not obvious what is going on or why we are doing it.